.travis.yml

```diff
@@ -9,6 +9,7 @@ git:
 cache:
   bundler: true
   cocoapods: true
+  podfile: ios/Podfile
```

Comment:
just asking here: if this is cached, what happens if we change something in that file? This will be recognized probably, right?

Reply:
I think the `pod install` command will simply notice that some files are outdated and overwrite them.
But if that doesn't happen, the build would probably fail, and you can go to https://travis-ci.org/github/mapswipe/mapswipe/caches and clear the cache for that branch, and rerun the CI job.